### ISSUE TITLE
Licensing issues?

### ISSUE THREAD

**Issue by dill** [karawoo/staRtrek#4]
As brought up in [this set of tweets](https://twitter.com/millerdl/status/527081247139725313) ("conversation", I guess), Memory Alpha [doesn't allow API access](http://en.memory-alpha.org/api/v1/). My interpretation of this is that it's to stop folks using the information in a product they sell (i.e. violating the "NC" part of [the license](http://creativecommons.org/licenses/by-nc/3.0/). Does this seem like your interpretation too?

In any case, I think we should attribute where the data comes from in each function.

Thoughts?

**Comment by karawoo** [comment]
Hrmmmm that is a good point. My understanding of licensing issues is pretty rudimentary but that interpretation sounds right to me. But I don't know how broad what's considered "commercial" is.

Do you think this means we need to change the package license to something else besides MIT?

**Comment by dill** [comment]
I think commercial is making 💵💷, which I don't think we're going to bring in...

I don't think we need to change the license as we're not licensing the data just our code.

(My understanding is only slightly less rudimentary than yours I think...)

**Comment by Gastove** [comment]
Yeah. My understanding is that this code (and the people writing it) would fall roughly under the heading of a "fan club" -- so as long as we don't claim any ownership over the ideas of Star Trek itself, I think we should be just fine.
